test/spec/modules/anPspParamsConverter_spec.js

```diff
@@ -51,7 +51,6 @@ describe('anPspParamsConverter', function () {
 
     const testBidderRequests = deepClone(bidderRequests);
 
-    debugger; //eslint-disable-line
```

Comment:
whopps

Reply:
ty

Reply:
yeah - sorry about that.  Finally caught it.